styles.css

```diff
@@ -0,0 +1,476 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
+
+:root {
+  --black: #272a31;
+  --orange: #ec5242;
+  --grey: #d3d3d3;
+  --mainFont: 'Lato', sans-serif;
+}
+
+html,
+body {
+  box-sizing: border-box;
+  font-family: var(--mainFont);
+  padding: 0;
+  margin: 0;
+  position: relative;
+  z-index: 10;
+}
+
+.display-none {
+  display: none;
+}
+
+.container {
+  max-width: 90%;
+  margin: 0 auto;
+}
+
+.modalcpst {
+  background-color: rgba(236, 82, 66, 0.9);
+  height: 100vh;
+  position: absolute;
+  z-index: 4;
+  width: 100%;
+}
+
+.modal__navigation ul {
+  list-style: none;
+  padding-left: 0;
+}
+
+.modal__navigation__li {
+  margin-bottom: 1.8rem;
+}
+
+.modal__navigation__li a {
+  color: var(--grey);
+  text-transform: capitalize;
+  font-size: 2.3rem;
+  text-decoration: none;
+}
+
+.modal__exit-button {
+  color: var(--grey);
+  display: flex;
+  justify-content: end;
+  padding: 25px 0;
+}
+
+.stop-scrolling {
+  overflow: hidden;
+}
+
+.topPageDiv {
+  display: none;
+}
+
+.topPage__img {
+  padding: 7px;
+}
+
+.topPage__text {
+  padding: 1px 5px;
+}
+
+.navbar {
+  display: none;
+}
+
+.hamburger {
+  padding: 10px;
+}
+
+.line {
+  width: 25px;
+  height: 5px;
+  background-color: var(--grey);
+  border-radius: 7px;
+  margin: 5px;
+}
+
+.logo {
+  width: 40px;
+}
+
+.nav-link {
+  color: var(--grey);
+}
+
+header {
+  background-image: url(./elements/bckgrnd8.jpeg);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  height: 100%;
+}
```

Comment:
- [ ] While you did a good job styling your project :1st_place_medal:, It is required to use the same background from the original design. [Here](https://ingdavidtm.github.io/capstone-biotechnology/images/backgroundHeader.png) you can find the required background.
- [ ] Also, kindly check the padding and margins of the content of this section because it seems not to be in the middle of your page as the template.

**Your design**

![Screenshot from 2023-02-09 06-30-52](https://user-images.githubusercontent.com/105117832/217801242-c5a87c8b-a353-43a4-9b57-7a48caee6d69.png)

**Template**

![Screenshot from 2023-02-07 04-46-32](https://user-images.githubusercontent.com/105117832/217210339-690f253b-f4a6-418d-81ba-b29dc4ab0b86.png)

Reply:
@IngDavidTM  with regards to the first comment on the use of different images I still don't understand why it's been flagged. I will change it though seeing as I now have the "required image".

Reply:
Now that is something that needs to be looked at is it possible to get fixed its